src/lib/statistics/statistics_objects/equal_distinct_count_histogram.cpp

```diff
@@ -49,11 +50,36 @@ template <typename T>
 std::vector<std::pair<T, HistogramCountType>> value_distribution_from_column(const Table& table,
                                                                              const ColumnID column_id,
                                                                              const HistogramDomain<T>& domain) {
-  // TODO(anybody) If you want to look into performance, this would probably benefit greatly from monotonic buffer
-  //               resources.
-  ValueDistributionMap<T> value_distribution_map;
-
+  // Since this function can be the main bottleneck for loading data in the benchmarks, we put some effort in
```

Comment:
Minor: `in the benchmark`, do we need this? Isn't you statement true for loading data in general?